src/comments.js

```diff
@@ -183,9 +183,19 @@ function printDanglingComments(path, options, sameIndent, filter) {
   return indent(concat([hardline, join(hardline, parts)]));
 }
 
+function hasLeadingComment(node) {
+  return node.comments && node.comments.some(comment => comment.leading);
+}
+
+function hasTrailingComment(node) {
```

Comment:
i think we had a "hadDanglingComments" in `comments.js` - should we move it here instead?

Reply:
@mgrip I think all related to comments better have in `comments.js` :smile:

Reply:
fine by me! should we move the `hasDanglingComments` helper in here then?